02-esc/challenges.js

```diff
@@ -184,18 +185,21 @@ function showAllChallenges(resultFromAPI) {
 		btnBook.classList.add("btnBook", "button", "primary");
 
 		btnBook.setAttribute("data-id", id);
+		btnBook.setAttribute("data-participants", minParticipants);
+		btnBook.setAttribute("data-participants", maxParticipants);
+
 
 		if (type == "onsite") {
-			btnBook.textContent = "Take challenge online";
-		} else {
 			btnBook.textContent = "Book this room";
+		} else {
+			btnBook.textContent = "Take challenge online";
 		}
 		challenge_item.appendChild(btnBook);
 
-		btnBook.addEventListener("click", function () {
-			// referens till bokningsfunktion
-		});
+		console.log(btnBook.getAttribute("data-id"));
```

Comment:
Console output is great for debugging, but we generally try to avoid including it in code that we share with others (e.g. check into git). Many projects actually forbid it, and automate checks to make sure `console.log()` is never included in code that is pushed to GitHub.

Reply:
Sorry for that, it was never my intention.